client/actions/loginActions.js

```diff
@@ -43,9 +37,7 @@ export function forgotPassword(body) {
             .then((response) => {
                 dispatch(forgotPasswordAction(response.data))
             })
-            .catch((err) => {
-                console.log("error ======>>>>>>", err);
-            })
+            .catch({/** */})
```

Comment:
please don't swallow exceptions.
use this instead
catch((err) => { console.error(err);}